include/vcpkg/base/delayed-init.h

```diff
@@ -7,23 +7,26 @@
 
 namespace vcpkg
 {
+    constexpr struct EmptyDelayedInit
+    {
+    } delayed_init_empty;
+
     // implements the equivalent of function static initialization for an object
     template<class T>
     struct DelayedInit
     {
+        DelayedInit() = default;
+        DelayedInit(EmptyDelayedInit) { }
```

Comment:
These ctors have the same effect; not sure at what you're getting at here.

Reply:
`delayed_init_empty` is a safe, copiable placeholder that can be passed through `.emplace()` functions.